CONTRIBUTING.md

```diff
@@ -324,14 +328,51 @@ Secretless supports using [Go plugins](https://golang.org/pkg/plugin/) to extend
 its functionality. To learn about writing new Secretless plugins and for more
 information on the types of plugins we currently support, visit the [plugin API directory](pkg/secretless/plugin).
 
-### Submodules
+## Submodules
 
 Secretless makes use of some plugins using [Git Submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules). 
 Development on submodules is similar to just working with a second repository, in that you can `cd` into it and 
 check out branches or make seperate commits. However, you also have the ability to commit and push recursively 
 from the parent repository. For help with this, it is recommended to review the "Publishing Submodule Changes" 
```

Comment:
Can you add a link to this here?

Reply:
It's all on one page, so I'm not sure how to link to that particular section of the doc :/

Reply:
I would just link to the page https://git-scm.com/book/en/v2/Git-Tools-Submodules and reference the section (as you're already doing)

Reply:
Better that than to leave people having to guess which page you're talking about